validation-test/IDE/crashers_2_fixed/rdar126923558.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-ide-test -code-completion -code-completion-token=COMPLETE -source-filename=%s
+
+actor Foo {
+  func bar() {}
+}
+
+func test(foo: isolated Foo) {
+  foo.#^COMPLETE^#
+}
```

Comment:
This test case doesn't have any closures, does it trigger the crash without the fix?

Reply:
Oh, yes. Indeed. I was trying other things in there and then forgot to reset it. Thanks for noticing.